### PR TITLE
Implement Overview page and sidebar component

### PR DESCRIPTION
**Pull request by hochiw**
## Features

- Removed HomeScreen
- Implemented Icons, IconButton, Divider helper components
- Implemented NavigationMenu and NavigationMenuItems
- Created WithSideBar wrapper to handle pages with sidebar
- Implemented OverviewScreen and integrate WithSideBar

## Screenshots
| Desktop | Mobile Menu Close | Mobile Menu Opened |
|---------|---------------------|-----------------------|
|Screenshot 2022-05-18 at 7 31 32 PM|Screenshot 2022-05-18 at 7 32 00 PM|Screenshot 2022-05-18 at 7 32 07 PM|

refs #33

### CODE REVIEW

**Comment by chihimng** [comment]
plz align mobile margin with design

File: react-app/src/components/OverviewScreen/OverviewScreen.tsx [hunk 1]
```diff
@@ -22,70 +15,11 @@ const OverviewScreen: React.FC = () => {
...
-            className={cn("hidden", "sm:flex")}
-          />
-        </div>
-        <div className={cn("flex-1", "bg-blue-400", "max-w-screen-2xl")}>
+      <WithSideBar>
+        <div className={cn("flex-1", "bg-white", "rounded-lg")}>
           <p>Main Content</p>
         </div>
-      </div>
+      </WithSideBar>
```
Comment (hochiw) [inline react-app/src/components/OverviewScreen/OverviewScreen.tsx:22]: should handle using router

**Comment by chihimng** [comment]
@hochiw ui nitpick on mobile:

- best to have bottom padding for the button list / menu in case scroll is needed:

image

- on click current tab, should dismiss the menu on mobile?

**Review by chihimng — CHANGES_REQUESTED** [review]

File: react-app/src/components/AppSideBar/AppSideBar.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    setIsMenuActive(!isMenuActive);
+  }, [isMenuActive]);
+
+  return (
+    <div
+      className={cn(
+        "h-full",
+        "w-full",
+        "overflow-auto",
+        "bg-gradient-to-b",
```
[the remark this answers is not in the document]
Comment (hochiw) [reply react-app/src/components/AppSideBar/AppSideBar.tsx:43]: this?

Screenshot 2022-05-23 at 2 18 42 PM

[the remark this answers is not in the document]
Comment (hochiw) [reply react-app/src/components/AppSideBar/AppSideBar.tsx:43]: from the most outer component for overview page in figma

**Comment by hochiw** [comment]
@chihimng updated :adore2x:

File: react-app/src/components/AppSideBar/AppSideBar.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (chihimng) [reply react-app/src/components/AppSideBar/AppSideBar.tsx:43]: not noticing this on screen @.@

**Review by chihimng — APPROVED** [review]

**Pull request merged**